setup.cfg

```diff
@@ -9,17 +9,6 @@ upload = upload upload_docs --upload-dir build/sphinx/html
 [bdist_rpm]
 doc-files = doc
 
-[nosetests]
```

Comment:
What I meant (just in case it wasn't clear): you need to figure out how to generate a coverage report with py.test.